lib/clients/rtm/client.js

```diff
@@ -403,13 +415,15 @@ RTMClient.prototype._handleWsMessageViaEventHandler = function _handleWsMessageV
   messageType, message) {
   var replyTo;
 
+  if (!isUndefined(this.dataStore)) {
+    this.dataStore.handleRtmMessage(this.activeUserId, this.activeTeamId, messageType, message);
+  }
+
```

Comment:
Hmm, thanks. I think it may be a misunderstanding on my part about how the `reply_to` ack is handled. Looking into that now.